src/rapids_dependency_file_generator/rapids_dependency_file_generator.py

```diff
@@ -329,32 +333,33 @@ def make_dependency_files(
     """Generate dependency files.
 
     This function iterates over data parsed from a YAML file conforming to the
-    `dependencies.yaml file spec <https://github.com/rapidsai/dependency-file-generator#dependenciesyaml-format>__`
+    `dependencies.yaml file spec <https://github.com/rapidsai/dependency-file-generator#dependenciesyaml-format>`_
     and produces the requested files.
 
-    Parameters
-    ----------
-    parsed_config : Config
-        The parsed dependencies.yaml config file.
-    file_keys : list[str]
-        The list of file keys to use.
-    output : set[Output]
-        The set of file types to write.
-    matrix : dict[str, list[str]] | None
-        The matrix to use, or None if the default matrix from each file key
-        should be used.
-    prepend_channels: list[str]
-        List of channels to prepend to the ones from parsed_config.
-    to_stdout : bool
-        Whether the output should be written to stdout. If False, it will be
-        written to a file computed based on the output file type and
-        config_file_path.
-
-    Raises
-    -------
-    ValueError
-        If the file is malformed. There are numerous different error cases
-        which are described by the error messages.
+    :param parsed_config: The parsed dependencies.yaml config file.
```

Comment:
This changed away from NumPy style. We use NumPy style docstrings in most of RAPIDS. I know it’s a lot of work to switch but we should just be consistent and use NumPy style.

Reply:
If there's a lot you can try using `pyment` or `docconvert` to automate the conversion.